src/theme/_variables.scss

```diff
@@ -1,15 +1,48 @@
 // Colors
+$white-text: #fcffff;
+$active-buttons: #f7a400;
+$hover-buttons: #fb7812;
+
+$background-one: #fff5ee;
+$background-two: #a6b7a9;
+$background-reserved: #457b6d;
+$header: #114a3e;
```

Comment:
Header should be 80% transparent

Reply:
> Header should be 80% transparent

ok. I used rgb colors 
rgb(17, 74, 62, 80)